demos/browser/server.js

```diff
@@ -19,25 +19,39 @@ console.info('Using index path', indexPagePath);
 
 const indexPage = fs.readFileSync(indexPagePath);
 
-// Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
+// Create ans AWS SDK Chime object. Region 'us-east-1' is globally available..
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
 const chime = new AWS.Chime({ region: 'us-east-1' });
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
```

Comment:
Do you think we could just use ChimeSDKMeeting but just hard code it the region to `us-east-1` like this?